src/components/ProgressReport/RatingModal/index.tsx

```diff
@@ -61,23 +60,26 @@ const ProgressReportRatingModal: FC<IProgressReportRatingModal> = (props) => {
 						theme={theme}
 					/>
 				</p>
-				<p
-					className='m-0 cursor-pointer p-0 text-sm font-normal text-pink_primary'
-					onClick={() => {
-						router.push(`/referenda/${postData?.postIndex}?tab=evaluation`);
+				<Link
```

Comment:
if it is a link, it shouldn't have e.preventDefault and stopPropagation, if it is a modal it should be a button and not a link